src/main/resources/templates/fragments.html

```diff
@@ -8,16 +8,18 @@
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
 
    <link rel="stylesheet" th:href="@{/styles.css}">
-
 </head>
+<header>
+   <img th:src="@{/images/treasureTask-header.jpg}" alt="Header Image" width="200" height="150">
+</header>
 <body>
 
 <header th:fragment="header">
```

Comment:
```suggestion
<header th:fragment="header">
   <img th:src="@{/images/treasureTask-header.jpg}" alt="Header Image" width="200" height="150">
```

Reply:
The img tag looks right to me but the image isn't showing. Maybe need another eye on this @lacourd